v22.1/row-level-ttl.md

```diff
@@ -0,0 +1,323 @@
+---
+title: Batch Delete Expired Data with Row-Level TTL
+summary: Automatically delete rows of expired data older than a specified interval.
+toc: true
+keywords: ttl,delete,deletion,bulk deletion,bulk delete,expired data,expire data,time to live,row-level ttl,row level ttl
+docs_area: develop
+---
+
+{% include {{page.version.version}}/sql/row-level-ttl.md %}
+
+By using Row-Level TTL, you can avoid the complexity of writing and managing scheduled jobs from the application layer to mark rows as expired and perform the necessary deletions. Doing it yourself can become complicated due to the need to balance the timeliness of the deletions vs. the potentially negative performance impact of those deletions on foreground traffic from your application.
```

Comment:
I would add this is a feature in preview mode - it may not satisfy production requirements due to some of the performance issues that may occur in large volume data sets (subject to feedback from @vy-ton)

Reply:
I added that it is in *beta*, along with your performance comment, and linked to the (updated) limitations section which now mentions some performance things